base/promotion.jl

```diff
@@ -251,7 +251,7 @@ end
 """
     ^(x, y)
 
-Exponentiation operator.
+Exponentiation operator. If `x is a matrix, computes matrix exponentiatin.
```

Comment:
I'd actually reword the second sentence as, "If `x` is a matrix, computes the matrix exponential."